content/configuration/environment.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Environment
+contributors:
+  - simon04
+---
+
+Eventually you will find the need to disambiguate in your `webpack.config.js` between [development](/guides/development) and [production builds](/guides/production-build). You have (at least) two options:
+
+## Using `--env`
+
+The webpack CLI support specifying build environment keys via `--cli` such as `--env.production` or `--env.platform=web`. To make use of those settings, change the configuration object into a function in `webpack.config.js`:
+
+```diff
+-module.exports = {
++module.exports = function(env) {
++  return {
+    plugins: [
+      new webpack.optimize.UglifyJsPlugin({
```

Comment:
Sorry, why does this need to be in Uglify?

```
    new webpack.DefinePlugin({
      'process.env.NODE_ENV': JSON.stringify('production')
    })
```
This way is referenced in https://github.com/webpack/webpack.js.org/blob/master/content/guides/production.md#node-environment-variable